src/components/__tests__/StatsInnerList-test.tsx

```diff
@@ -37,7 +37,7 @@ describe("StatsInnerList", () => {
     let newData = {...data, ...{"cancelled": []}};
     wrapper.setProps({ data: newData });
     let categories = wrapper.find(".stats-category");
-    ["Production", "Testing", "Cancelled"].forEach((name, idx) => {
+    ["Production", "Testing"].forEach((name, idx) => {
```

Comment:
Why the removal of "Cancelled" in the test?

Reply:
I changed the functionality slightly.  Old behavior: a `.stats-category` bar would be rendered for each of the three stages, even if there weren't any libraries under that category.  This looked kind of silly (and potentially confusing); I had been meaning to change it for a while, and this was a good opportunity.  New behavior: the `.stats-category` bar is only rendered if there are libraries to show.  TLDR: in `newData` in this test, the "cancelled" array is empty, so the interface shouldn't display a category bar that says "Cancelled."